action-core/src/main/java/com/adyen/checkout/action/core/internal/ui/DefaultGenericActionDelegate.kt

```diff
@@ -129,6 +132,12 @@ internal class DefaultGenericActionDelegate(
             observeViewFlow(delegate)
         }
 
+        val event = GenericEvents.action(
```

Comment:
One concern: if merchants use an action component directly to handle the action, then we would not send any event. Maybe we should move this event to each specific action delegate? WDYT?

Reply:
Hmm, indeed it makes sense to move them to concrete action delegates. Will do it, thanks for the catch!